xbmc/guilib/GUIButtonControl.h

```diff
@@ -32,12 +32,15 @@
 #include "GUILabel.h"
 #include "GUIControl.h"
 
+class CGUISettingsSliderControl;
+
 /*!
  \ingroup controls
  \brief
  */
 class CGUIButtonControl : public CGUIControl
 {
+  friend class CGUISettingsSliderControl;
```

Comment:
@FernetMenta 
I needed quick acces to `m_Label`. Would it make sense to create a new public method to access m_Label. It is needed to properly place the labels at a correct position and size.